deploy/roles/citus/tasks/main.yml

```diff
@@ -34,3 +34,26 @@
     mode: "0600"
   tags:
     - citus
+
+- name: Ensure systemd directory exists
+  file:
+    state: directory
+    path: /etc/systemd/system/postgresql-9.6.service.d
```

Comment:
Was this path not present when Postgres 9.6 was installed?  It should have already been there, just want to verify.

Reply:
Although I could be wrong, I'm still getting familiar with the inner workings of SystemD, it's possible this directory is only created if needed.

Reply:
I thought so too, but it wasn't there and my job failed. There's probably a better way to force it to create it.